src/mobu/storage/git.py

```diff
@@ -0,0 +1,73 @@
+"""A very simple async replacement for GitPython, which has been
+abandoned and is distinctly not thread- or async-safe.
+"""
+
+from pathlib import Path
+
+from structlog.stdlib import BoundLogger
+
+from ..models.user import AuthenticatedUser
+from .process import Process
+
+
+class Git:
+    """A very basic async Git client based on asyncio.subprocess."""
+
+    def __init__(
+        self,
+        *,
+        repo: Path | None = None,
+        user: AuthenticatedUser | None = None,
+        logger: BoundLogger | None = None,
+    ) -> None:
+        self._user = user
+        if self._user is None:
+            self._user = AuthenticatedUser(
+                username="gituser",
+                uidnumber=60001,
+                gidnumber=60001,
+                scopes=[],
+                token="dummy token",
+            )
```

Comment:
All of this user stuff appears to be used only to add the username to the `AsyncProcessError` exception. I think you can delete all of it; the monkey already handles adding the user when reporting failures.